sgigli/splashzone-industry-dive
# Customizable News Post Ads

Solution Overview: In the advertising app, I created an Advertisement model, and then registered that model with the CMS admin. I added a one-to-many relationship between Advertisements and NewsPosts, so that a single ad could be set to multiple news posts. I then configured the Advertisement model to upload its logo to the media folder. And then in the NewsPost views and template, I selected and showed the post's advertisement if it was present, and defaulted to the Industry Dive house ad when it wasn't.

Steps to Test:
1. Load the database migrations with `python3 manage.py migrate`.
2. Login to the admin CMS.
3. Create an advertisement with a logo.
4. Add this advertisement to any news post.
5. Return to http://127.0.0.1:8000/ and click on the post.
6. Confirm that the ad appears at the top of the post.
7.  Confirm that the Industry Dive house ad appears at the top of any other posts.
